miso-web/src/main/webapp/scripts/box_visualization.js

```diff
@@ -30,6 +30,12 @@
 var BoxPosition = function(opts) {
   var self = {};
   
+  isMultiSelectClick = function(event) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "isMultiSelectClick" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2703)